### PR TITLE
feat: display clients details for other users

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

- opening DeviceScreen from OtherUserProfile
- additional changes to finish the feature 
- merged with `develop`
- updated kalium version to https://github.com/wireapp/kalium/pull/1544

https://user-images.githubusercontent.com/6539347/225886230-c311deb1-ecfd-45d2-8ad2-80d80b6bfcc2.mp4

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1559/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1559/3/) **failed**.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1559/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(6008990)(6008990)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(6008990)(6008990)-staging-compat.apk)

File: app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt [hunk 1]
```diff
@@ -157,14 +162,18 @@ private fun DeviceItemTexts(device: Device, placeholder: Boolean, isDebug: Boole
...
+        Text(
+            style = MaterialTheme.wireTypography.body02,
+            color = MaterialTheme.wireColorScheme.onBackground,
+            text = device.name,
+            modifier = Modifier
+                .wrapContentWidth()
+                .shimmerPlaceholder(visible = placeholder)
+        )
+        Spacer(modifier = Modifier.width(MaterialTheme.wireDimensions.spacing8x))
+        VerifyLabel(device.isVerified, Modifier.wrapContentWidth())
```
Comment (MohamadJaara) [inline app/src/main/kotlin/com/wire/android/ui/authentication/devices/DeviceItem.kt]: The same item is used when encountering too many clients after login and in that case, we do not need to display the VerifyLabel

**Review by MohamadJaara — CHANGES_REQUESTED** [review]
Looking great as always, we only need to not display the VerifyLabel on the too many clients screen

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1559/5/) **failed**.

**Review by MohamadJaara — APPROVED** [review]

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1559/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.0.2(6034975)(6034975)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.0.2(6034975)(6034975)-staging-compat.apk)

**Pull request merged**